internal/metastore/kv/rootcoord/suffix_snapshot.go

```diff
@@ -543,3 +553,80 @@ func (ss *SuffixSnapshot) MultiSaveAndRemoveWithPrefix(saves map[string]string,
 	}
 	return err
 }
+
+func (ss *SuffixSnapshot) Close() {
+	close(ss.closeGC)
+}
+
+// startBackgroundGC the data will clean up if key ts!=0 and expired
+func (ss *SuffixSnapshot) startBackgroundGC() {
+	log.Debug("suffix snapshot GC goroutine start!")
+	ticker := time.NewTicker(15 * time.Minute)
```

Comment:
15 min might be too frequent, let's make it 1 day

Reply:
1 day is a little bit too long for frequently creating and dropping collection scenarios, Maybe 1 hour is better?